.azure/pipelines/quarantined-pr.yml

```diff
@@ -0,0 +1,149 @@
+# We want to run quarantined tests on master as well as on PRs
+trigger:
+  batch: true
+  branches:
+    include:
+    - master
+
+schedules:
+- cron: "0 */4 * * *"
+  displayName: Every 4 hours test run
+  branches:
+    include:
+    - master
+  always: true
+
+variables:
+- ${{ if ne(variables['System.TeamProject'], 'internal') }}:
+  - name: _UseHelixOpenQueues
+    value: 'true'
+- ${{ if eq(variables['System.TeamProject'], 'internal') }}:
+  - group: DotNet-HelixApi-Access
+  - name: _UseHelixOpenQueues
+    value: 'false'
+
+jobs:
+- template: jobs/default-build.yml
+  parameters:
+    jobName: Helix_quarantined_x64
+    jobDisplayName: 'Tests: Helix'
+    agentOs: Windows
+    timeoutInMinutes: 240
+    steps:
+    # Build the shared framework
+    - script: ./build.cmd -ci -nobl -all -pack -arch x64 /p:ASPNETCORE_TEST_LOG_DIR=artifacts/log
+      displayName: Build shared fx
+    - script: ./build.cmd -ci -nobl -noBuildRepoTasks -restore -noBuild -noBuildNative -projects src/Grpc/**/*.csproj
+      displayName: Restore interop projects
+    - script: ./build.cmd -ci -nobl -noBuildRepoTasks -noRestore -test -all -noBuildJava -noBuildNative
+              -projects eng\helix\helix.proj /p:RunQuarantinedTests=true /p:IsRequiredCheck=true /p:IsHelixJob=true
+              /p:BuildInteropProjects=true /p:RunTemplateTests=true /p:ASPNETCORE_TEST_LOG_DIR=artifacts/log
+      displayName: Run build.cmd helix target
+      continueOnError: true
+      env:
+        HelixApiAccessToken: $(HelixApiAccessToken) # Needed for internal queues
+        SYSTEM_ACCESSTOKEN: $(System.AccessToken) # We need to set this env var to publish helix results to Azure Dev Ops
+    artifacts:
+    - name: Helix_logs
+      path: artifacts/log/
+      publishOnError: true
+
+- template: jobs/default-build.yml
```

Comment:
Everything from here down isn't needed in quarantined-tests.yml anymore.

But, quarantined tests.yml should run the Helix ARM64 tests and doesn't do that now. Need to copy the job starting at https://github.com/dotnet/aspnetcore/blob/524349beb4f5ad0da7bb4989247879e67bd623ad/.azure/pipelines/helix-matrix.yml#L45 there then change it slightly to run the quarantined tests on ARM64 agents.

Reply:
Its already in there https://github.com/dotnet/aspnetcore/blob/master/.azure/pipelines/quarantined-tests.yml#L49

Reply:
Up to you whether you want to make the quarantined-tests.yml changes in a separate PR.

Reply:
Ignore the "copy the job" part above. I wasn't looking at the 'master' copy of quarantined-tests.yml. Just need to remove the three existing non-Helix test jobs from that file.

Reply:
Cool I'll merge this as is and do a separate PR to clean up the other pipeline